src/app/chat/views/message_list.nim

```diff
@@ -100,6 +100,11 @@ QtObject:
       self.messages.add(message)
     self.endInsertRows()
 
+  proc clear*(self: ChatMessageList) =
+    self.beginRemoveRows(newQModelIndex(), self.messages.len, self.messages.len)
+    self.messages = @[]
+    self.endRemoveRows()
```

Comment:
This doesn't seem to work =/
While the message deletion seems to have been successful through the RPC call, I can't manage to get the messages removed from the UI.

If anyone knows how to reset the abstract list model and have it rerender accordingly, please let me know!

Reply:
try with this!
```
self.beginResetModel()
self.messages = @[]
self.endResetModel()
```